src/game/tile.rs

```diff
@@ -192,7 +194,6 @@ impl TileGraphic {
         }
         // two pixels around the corner
         for row_index in &[
-            1,
```

Comment:
what? the tiles look weird now cuz of this

Reply:
uh mb, fixed.